### DISCUSSION TITLE
求overgrowth

### DISCUSSION THREAD

**Discussion by yincolor** [Gamuxorg/bbs#376] [投递和请求游戏]
是一个3D 动作游戏，steam上卖得太贵啦

**Comment by shouhuanxiaoji** [comment]
处理中

**Comment by shouhuanxiaoji** [comment]
已添加

**Comment by yincolor** [comment]
感谢，这里边有中文吗？

发自 Outlook<http://aka.ms/weboutlook> 个人账户
________________________________
发件人: 小鸡 <notifications@github.com>
发送时间: 2018年12月10日 21:00
收件人: Gamuxorg/bbs
抄送: BCCatty; Author
主题: Re: [Gamuxorg/bbs] 求overgrowth (#34)


已添加

―

**Comment by shouhuanxiaoji** [comment]
并没有，看到3DM有不完全汉化，不过没有测试，你如果有兴趣可以试试
